Pod/Classes/BugShaker.swift

```diff
@@ -8,12 +8,16 @@
 import UIKit
 import MessageUI
 
+public protocol BugShakerDelegate {
+  func ShouldPresentReportPrompt() -> Bool
+}
 
 public class BugShaker {
-  
+
   struct Config {
     static var toRecipients: [String]?
     static var subject: String?
+    static var delegate: BugShakerDelegate?
```

Comment:
I tend to think of the `Config` struct as a collection of vars that will be used for the eventual email report. 

Could we move `delegate` to a `BugShaker` static class property?